### PR TITLE
Add timeout option to build parameters

### PR DESCRIPTION
**Pull request by daxmc99**
Addresses https://github.com/rancher/rio/issues/418

### CODE REVIEW

File: cli/cmd/create/create.go [hunk 1]
```diff
@@ -68,6 +69,7 @@ type Create struct {
 	Secret                 []string          `desc:"Secrets to inject to the service (format: name:target)"`
 	StageOnly              bool              `desc:"Whether to stage new created revision for build"`
 	T_Tty                  bool              `desc:"Allocate a pseudo-TTY"`
+	Timeout                int64             `desc:"Timeout for build, default to 10m" default:"10m"`
```
Comment (ibuildthecloud) [inline cli/cmd/create/create.go]: Should be `BuildTimeout`

**Comment by MarkBennett** [comment]
Let me know if you need a tester @daxmc99!

File: cli/cmd/attach/attach.go [hunk 2]
```diff
@@ -10,7 +10,7 @@ import (
 )
 
 type Attach struct {
-	Timeout string `desc:"Timeout waiting for the container to be created to attach to" default:"1m"`
+	Timeout string `desc:"BuildTimeout waiting for the container to be created to attach to" default:"1m"`
```
Comment (StrongMonkey) [inline cli/cmd/attach/attach.go]: :-) don't use refactor method in goland

Comment (daxmc99) [reply cli/cmd/attach/attach.go]: I see that now :(

**Pull request merged**